admin-ui/plugins/auth-server/redux/features/authNLdapSlice.js

```diff
@@ -0,0 +1,84 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+const initialState = {
+  ldapList: [],
+  loading: false,
+  saveOperationFlag: false,
+  errorInSaveOperationFlag: false,
+}
```

Comment:
Initialize missing state.item to avoid undefined reads

setCurrentItem writes state.item but initialState lacks it. Initialize explicitly.


```diff
 const initialState = {
   ldapList: [],
   loading: false,
   saveOperationFlag: false,
   errorInSaveOperationFlag: false,
+  item: null,
 }
```